src/components/MenuItem.js

```diff
@@ -59,7 +59,7 @@ const MenuItem = props => (
     >
         {({hovered, pressed}) => (
             <>
-                <View style={[styles.flexRow, styles.pointerEventsNone]}>
+                <View style={[styles.flexRow, styles.pointerEventsNone, styles.w100, styles.pr5]}>
```

Comment:
This padding will break the UI.

```suggestion
                <View style={[styles.flexRow, styles.pointerEventsNone, styles.flex1]}>
```